mcs/class/test-helpers/PlatformDetection.cs

```diff
@@ -3,9 +3,10 @@ namespace System
 	static partial class PlatformDetection
 	{
 		public static readonly bool IsNetNative = false;
-
+		public static readonly bool IsNotWinRT = true;
+		public static readonly bool IsWinRT = false;
 		public static readonly bool IsFullFramework = true;
-
-		public static bool IsNonZeroLowerBoundArraySupported => true;
```

Comment:
IsNonZeroLowerBoundArraySupported has to be a property, corefx testing framework depends on that